### PR TITLE
Add more prime_number tests in test_script

### PR DESCRIPTION
**Pull request by dannydav709**
Main branch & feature/new_tests_2 branch

(features/new_tests branch was already merged to main branch. feature/new_tests_2 just contains a few more tests to run in the test_script.py file.)

### CODE REVIEW

File: practice_project/test_script.py [hunk 1]
```diff
@@ -9,6 +9,10 @@ def test_prime_numbers():
     assert is_prime(17) == True
     assert is_prime(19) == True
     
+    assert is_prime(23) == True
+    assert is_prime(29) == True
+
+
```
Comment (leonstaubach) [inline practice_project/test_script.py:17]: Good! I would just add tests for some larger numbers and edge-cases (negative numbers, non-number input)

**Review by leonstaubach — APPROVED** [review]
